src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm.tsx

```diff
@@ -55,34 +55,16 @@ class RuleForm extends React.Component<Props> {
                 placeholder: t('My Incident Rule Name'),
                 required: true,
               },
-              {
-                name: 'includeAllProjects',
-                type: 'boolean',
-                label: t('Apply to all Projects'),
-                help: t(
-                  'This should apply to all projects as well as all future projects'
-                ),
-              },
```

Comment:
Will we be removing the API features and models related to excluded projects and the many to many relationship between rules and projects?

Reply:
No, API is staying the same so that we can support them in the future.